src/boulder-guide.erb

```diff
@@ -3,7 +3,45 @@ layout: page
 title: Boulder Guide
 top_level: false
 ---
+<% 
```

Comment:
For simplicity, I placed the array of links/suggestions here, but please let me know if they should be moved elsewhere